srtcore/sync.h

```diff
@@ -174,6 +174,9 @@ Duration<steady_clock> seconds_from(int64_t t_s);
 
 inline bool is_zero(const TimePoint<steady_clock>& t) { return t.is_zero(); }
 
+timespec us_to_timespec(const uint64_t time_us);
+
+
```

Comment:
Declaration of `us_to_timespec()` is not needed. Only used inside the sync.cpp

Reply:
Right. Can be static.